velox/docs/functions/presto/conversion.rst

```diff
@@ -919,6 +979,54 @@ Invalid example
   SELECT cast('-3E+2.1' as decimal(12, 2)); -- Value is not a number
   SELECT cast('3E+' as decimal(12, 2)); -- Value is not a number
 
+Cast to IPAddress
```

Comment:
Cast to IPADDRESS

See https://facebookincubator.github.io/velox/functions/presto/conversion.html#cast-to-timestamp